launch.rb

```diff
@@ -110,6 +110,11 @@ def self.register_available_controllers
       FastlaneCI::FastlaneApp.use(FastlaneCI::ProviderCredentialsController)
       FastlaneCI::FastlaneApp.use(FastlaneCI::UsersController)
       FastlaneCI::FastlaneApp.use(FastlaneCI::BuildController)
+
+      # Load JSON controllers
+      require_relative "features-json/project_json_controller"
```

Comment:
I like have you have this space down here, it should help with avoiding merge conflicts 😄